packages/playwright/src/worker/workerMain.ts

```diff
@@ -330,8 +330,8 @@ export class WorkerMain extends ProcessRunner {
       testInfo._floatingPromiseScope.clear();
     };
 
-    await testInfo._runAsStage({ title: 'setup and test' }, async () => {
-      await testInfo._runAsStage({ title: 'start tracing', runnable: { type: 'test' } }, async () => {
+    await (async () => {
```

Comment:
Replace this anonymous function call with `try {} catch {}` as that's the only thing it does now.

Reply:
I tried that 😄 However, there are early returns from this function, so I had to keep it a function. I can try to improve things in a follow-up.